benchmarks/serialize.py

```diff
@@ -0,0 +1,94 @@
+from contextlib import contextmanager
```

Comment:
(I'm planning to remove this file from the final PR, but wanted to include it in case you wanted to independently confirm the benchmark results)